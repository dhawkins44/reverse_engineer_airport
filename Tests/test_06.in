4 3 45 35
0 2
0 0
1 1
2 1
3 1
0 3 35
1 2 45
1 3 45