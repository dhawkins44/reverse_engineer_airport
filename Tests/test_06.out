Verify Path: 1
125: 0 3 1 2 
